ionic2/src/theme/custom-variables.scss

```diff
@@ -0,0 +1,56 @@
+/* constants */
```

Comment:
Same here, I think it would be good to delete the old one as we bring this one in to this structure.